c++/src/kj/mutex.h

```diff
@@ -236,6 +238,8 @@ class Mutex {
 #elif _WIN32 || __CYGWIN__
     uintptr_t condvar;
     // Actually CONDITION_VARIABLE, but don't want to #include <windows.h> in header.
+#elif defined(KJ_NO_THREADS)
+    uint32_t dummy_condvar;
```

Comment:
Condvars aren't actually implemented (and can't be implemented) in the absence of threads, so should the whole condvar interface (`wait()` / `when()`) be `#if`ed out in no-threads mode?